Voycar.Api.Web.Tests.Unit/Users/Get/Verify/Endpoint.cs

```diff
@@ -67,11 +67,10 @@ public async Task Verify_User_Fails_And_Returns_BadRequest()
             .Returns((User?)null);
 
         // Act
-        await ep.HandleAsync(this.Request, default);
-        var rsp = ep.HttpContext.Response;
+        var exception = await Assert.ThrowsAsync<ValidationFailureException>(() => ep.HandleAsync(this.Request, default));
 
         // Assert
-        Assert.NotNull(rsp);
-        Assert.Equal(StatusCodes.Status400BadRequest, rsp.StatusCode);
+        Assert.NotNull(exception);
+        Assert.Equal("ThrowError() called! - Token does not belong to any user", exception.Message);
```

Comment:
So jetzt hab' ichs aber richtig. Credit: https://ardalis.com/testing-exceptions-with-xunit-and-actions/
```suggestion
        // Act - local function
        async Task Act() => await ep.HandleAsync(this.Request, default);

        // Assert
        var exception = await Assert.ThrowsAnyAsync<ValidationFailureException>(Act);
        Assert.NotNull(exception);
        Assert.Equal("ThrowError() called! - Token does not belong to any user", exception.Message);
```
So würde ich die anderen Endpunkte dann auch gerne machen.